types/yaireo__tagify/index.d.ts

```diff
@@ -92,12 +101,25 @@ declare namespace Tagify {
          */
         includeSelectedTags: boolean;
 
+        /**
+         * Escape HTML entities in the suggestions' rendered text.
+         * @default true
+         */
+        escapeHTML: boolean;
+
         /**
          * Controls where the dropdown menu is positioned.
          * @default 'all'
          */
         position: DropDownPosition;
 
+        /**
+         * Dictates the dropdown's horizontal starting position. By default,
+         * it would be aligned with the left side of the Tagify component.
+         * @default false
+         */
+        RTL: boolean;
+
```

Comment:
I took another look at the tagify docs and found one more property. Could you also add the `dropdown.sortby`? If I read the docs correctly, it should be something like

```ts
/**
 * If set to `startsWith`, the suggestions list will be sorted with matched items which starts with
 * the query shown first, and exact matches shown before all.
 * 
 * If this setting is defined as a function, it receives the filtered items and the query and
 * it must return the sorted items.
 * 
 * (default sorting order is same as the whitelist's)
 */
sortby: "startsWith" | ((items: T[], query: string) => T[]);
```

Reply:
How did we miss that one? It has been already introduced 2 years ago. Good catch! I just added this one as well.